Testnet/Ticketbooth/Ticketbooth/TicketContract_1_0_0.cs

```diff
@@ -0,0 +1,447 @@
+﻿using Stratis.SmartContracts;
+
+public class TicketContract_1_0_0 : SmartContract
+{
+    /// <summary>
+    /// Creates a new ticketing contract
+    /// </summary>
+    /// <param name="smartContractState"></param>
+    /// <param name="seatsBytes">The serialized array of seats</param>
+    /// <param name="venueName">The venue that hosts the contract</param>
+    public TicketContract_1_0_0(ISmartContractState smartContractState, byte[] seatsBytes, string venueName)
+        : base(smartContractState)
+    {
+        var seats = Serializer.ToArray<Seat>(seatsBytes);
+        var tickets = new Ticket[seats.Length];
```

Comment:
For what we discussed earlier, might be best to assert a max length here of seats. Would want to make sure that the max amount of seats allowed won't break the contract in any methods that loop through them hitting gas limits.

Reply:
Updated this, now there's a maximum limit